native/cocos/ui/edit-box/EditBox-ios.mm

```diff
@@ -259,86 +191,55 @@ CGRect getSafeAreaRect() {
     return viewRect;
 }
 
-void addTextInput(const cc::EditBox::ShowInfo &showInfo) {
-    auto safeAreaRect = getSafeAreaRect();
-    int height = getTextInputHeight();
-    CGRect rect = CGRectMake(safeAreaRect.origin.x,
-                             safeAreaRect.size.height - height,
-                             safeAreaRect.size.width,
-                             height);
-    if (showInfo.isMultiline)
-        initTextView(safeAreaRect, rect, showInfo);
-    else
-        initTextField(rect, showInfo);
-
-    UIView *textInput = getCurrentView();
-    UIView *view = UIApplication.sharedApplication.delegate.window.rootViewController.view;
-    [view addSubview:textInput];
-    [textInput becomeFirstResponder];
+//TODO: Get hash with different type of showInfo, for example different inputAccessoryView
+NSString *getHash(const cc::EditBox::ShowInfo* showInfo) {
+    return showInfo->isMultiline?@"textView":@"textField";
```

Comment:
Currently,  getHash is not a real hash computation function. It's better to rename it to some thing like `getTextInputType`.

Reply:
1